requirements-osbs.txt

```diff
@@ -169,8 +169,6 @@ Cython==0.29.23
 Pygments==2.9.0
```

Comment:
I'm not sure we need this whole section; `pygments, matplotlib, ...`

Reply:
good catch, remove the section along with some other test related deps spread throughout the file.